### ISSUE TITLE
Transformer overfits early under BASE settings

### ISSUE THREAD

**Issue by xuanqing94** [tensorflow/models#5573]
### System information
- **What is the top-level directory of the model you are using**: official/transformer
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: No
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Ubuntu 16.04
- **TensorFlow installed from (source or binary)**: binary
- **TensorFlow version (use command below)**: 1.10
- **CUDA/cuDNN version**: 9.2
- **GPU model and memory**: 1080Ti
- **Exact command to reproduce**: Follow the training bash code in README.md

You can collect some of this information using our environment capture script:

### Describe the problem
I follow the code in the README of transformer, but it shows that the BLEU score quickly drops and never reached the claimed score (base=27.x) see the screen shot below:

https://drive.google.com/file/d/1npnY6yyMsDiBDz5UZD0IThbCn2HwEhG7/view?usp=sharing

Anyone has successfully replicated the result?

**Comment by ywen666** [comment]
I came across the same issue.

**Comment by wt-huang** [comment]
You can try to increase batch size, decrease learning rate or use more GPUs. 
max_length could help but only to some extent. 
Leave it running more training steps to see if the BLEU drop continues.
